README.md

```diff
@@ -127,8 +136,8 @@ SAML_ATTRIBUTE_MAPPING = {
 ````
 
 
-Tests
------
+Running tests
+-------------
 
```

Comment:
Here we should say "Only for developer", that's something to be run only in the example project and not in a custom project